### DISCUSSION TITLE
Index names

### DISCUSSION THREAD

**Discussion by musicpanda** [PrestaShop/PrestaShop#37729] [Ideas]
### The problem

Recently I see that Prestashop is adopting random index names like IDX_9811C64067A664FB for its indexes. Until now it concerns a limited number of tables.

I don't like that. It makes it hard to know what an index is about or to test for the presence of an index in an old shop.

What is the reason for this?

### The solution

I would like to see a return to normal index names.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
> Recently I see that Prestashop is adopting random index names like IDX_9811C64067A664FB for its indexes. Until now it concerns a limited number of tables.

Hello, can you provide an URL about what you talk? I checked https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/data/db_structure.sql and https://github.com/PrestaShop/autoupgrade/blob/dev/upgrade/sql/9.0.0.sql did not see it

**Comment by musicpanda** [comment]
Maybe it is just the shops I am working with at the moment. So I am mistaken.
